doc/UsersGuide/source/BuildingAndRunning.rst

```diff
@@ -0,0 +1,199 @@
+   | **EnvironmentVariable** | **Description**                              | **Hera Intel Value** |
+   +=========================+==============================================+======================+
+   |  CMAKE_C_COMPILER       | Name of C compiler                           | mpiicc               |
+   +-------------------------+----------------------------------------------+----------------------+
+   |  CMAKE_CXX_COMPILER     | Name of C++ compiler                         | mpiicpc              |
+   +-------------------------+----------------------------------------------+----------------------+
+   |  CMAKE_Fortran_COMPILER | Name of Fortran compiler                     | mpiifort             |
+   +-------------------------+----------------------------------------------+----------------------+
+   |  CMAKE_Platform         | String containing platform and compiler name | hera.intel           |
+   +-------------------------+----------------------------------------------+----------------------+
+
+If you are not running on one of the pre-configured platforms, you will need to set the environment variables
+in a different way.
+
+If you used one of the platform- and compiler-specific ``README`` files in the ``doc/`` directory of NCEPLIBS-external
+to build the prerequisite libraries, there is a script in the ``NCEPLIBS-ufs-v1.1.0/bin`` directory called
+``setenv_nceplibs.sh`` that will set the NCEPLIBS-external variables for you.
+
+Of course, you can also set the values of these variables yourself if you know where the paths are on your system.
+
+--------------------------------------------
+Setting the CCPP_SUITES environment variable
+--------------------------------------------
+
+In order to have one or more CCPP physics suites available at runtime, you need to select those suites at
+build time by setting the ``CCPP_SUITES`` environment variable. Multiple suites can be set, as shown below
+in an example for the bash shell:
+
+.. code-block:: console
+
+    export CCPP_SUITES="FV3_GFS_v15p2,FV3_GFS_v16beta"
+
+If ``CCPP_SUITES`` is not set, the default is set to ``‘FV3_GFS_v15p2’`` in ``build.sh``.
+
+------------------
+Building the model
+------------------
+The UFS Weather Model uses the CMake build system.  There is a build script called ``build.sh`` in the
+top-level directory of the WM repository that configures the build environment and runs the ``make``
+command.  This script also checks that all necessary environment variables have been set.
+
+If any of the environment variables have not been set, the ``build.sh`` script will exit with a message similar to:
+
+.. code-block:: console
+
+   ./build.sh: line 11: CMAKE_Platform: Please set the CMAKE_Platform environment variable, e.g. [macosx.gnu|linux.gnu|linux.intel|hera.intel|...]
+
+The WM can be built by running the following command from the `ufs-weather-model` directory:
+
+.. code-block:: console
+
+   ./build.sh
```

Comment:
This is not true.
The model will fail to compile because it will try to build all the CCPP suites and that returns an internal compiler error.

Reply:
That is unless `export CCPP_SUITES='some_valid_suite(s)`